frontend/src/routes.ts

```diff
@@ -1,3 +1,17 @@
+export enum OrgTab {
+  Dashboard = "dashboard",
+  Workflows = "workflows",
+  Items = "items",
+  Collections = "collections",
+  BrowserProfiles = "browser-profiles",
+  Settings = "settings",
+}
+
+export enum RouteNamespace {
+  PrivateOrgs = "orgs",
+  PublicOrgs = "profile",
```

Comment:
This can easily be changed if someone has a pithy and creative idea for a public namespace.